Description: Don't hardcode optimization level to -O0.
Author: Daniel Svensson <dsvensson@gmail.com>
Forwarded: not-needed

--- a/wscript
+++ b/wscript
@@ -16,8 +16,6 @@ def configure(conf):

 	conf.check_vala((0, 24, 0))

-	conf.env.append_unique("CFLAGS", ["-g", "-O0"])
-
 	conf.check_cfg(package='gio-2.0', atleast_version='2.40', args='--cflags --libs')
 	conf.check_cfg(package='gio-unix-2.0', atleast_version='2.40', args='--cflags --libs')
 	conf.check_cfg(package='glib-2.0', atleast_version='2.40', args='--cflags --libs')
